tests/test_client.py

```diff
@@ -279,6 +280,7 @@ def test_get_homegraph(
         self.assertEqual(m_get_home_graph_request.call_count, 1)
 
         # Expired homegraph
+        assert self.client.homegraph_date is not None
```

Comment:
Let's have here `unittest` pattern, just to have it consistent with the rest of the code?
```suggestion
        self.assertIsNotNone(self.client.homegraph_date)
```

Reply:
Unfortunately, `mypy` can't narrow type from such calls: https://github.com/python/mypy/issues/4063